boards/riscv32/hifive1/hifive1_defconfig

```diff
@@ -16,4 +16,6 @@ CONFIG_GPIO=y
 CONFIG_GPIO_SIFIVE=y
 CONFIG_SPI=y
 CONFIG_SPI_SIFIVE=y
+CONFIG_PWM=y
+CONFIG_PWM_SIFIVE=y
```

Comment:
Are you sure you always want to enable PWM?

Reply:
I have some inkling of "by default, enable all drivers" as being some kind of guideline. Could be wrong there.

Reply:
I think we are going with the practice that the app would enable CONFIG_PWM and in the board Konfig.defconfig you'd do:

```
if PWM

config PWM_SIFIVE
	def_bool y
```

something like that, might also be at the SoC.defconfig level instead of board.

Reply:
I also added an `#if defined(CONFIG_PWM)` around the PWM pinmux enables for the HiFive 1.